config.js

```diff
@@ -7,6 +7,8 @@ var config = {};
 config.webPort = process.env.WEB_PORT || 8000;
 config.serverVersion = '4.0.136';
 config.apiVersion = '4.0.7';
+//config.interface = 'Wi-Fi' //Windows Name
+//config.interface = 'wlan0' //linux name
```

Comment:
This should be done like all the other user configurable presets:
config.interface = process.env.WEB_INTERFACE || null;

Then the user can set the interface in his .env file with:
WEB_INTERFACE = "wlan0"